.github/workflows/smoke-test.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       - uses: AriPerkkio/eslint-remote-tester-run-action@v3
         with:
           issue-title: 'Results of weekly scheduled smoke test'
-          eslint-remote-tester-config: smoke-test/eslint-remote-tester.config.js
+          eslint-remote-tester-config: smoke-test/eslint-remote-tester.config.ts
```

Comment:
@G-Rath this path is still incorrect 😄 

```diff
-          eslint-remote-tester-config: smoke-test/eslint-remote-tester.config.ts
+          eslint-remote-tester-config: eslint-remote-tester.config.ts
```

https://github.com/jest-community/eslint-plugin-jest/issues/1002#issuecomment-1003642151

Reply:
Yup